sql/core/src/test/scala/org/apache/spark/sql/execution/BaseScriptTransformationSuite.scala

```diff
@@ -654,6 +654,23 @@ abstract class BaseScriptTransformationSuite extends SparkPlanTest with SQLTestU
         df.select($"ym", $"dt").collect())
     }
   }
+
+  test("SPARK-36675: TRANSFORM should support timestamp_ntz (no serde)") {
+    withTempView("v") {
```

Comment:
Sorry, I haven't found where it is used.

Reply:
Oops. I just forgot to remove this.